packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.controller.js

```diff
@@ -71,4 +76,44 @@ export default class PciBlockStorageController {
   refreshMessages() {
     this.messages = this.messageHandler.getMessages();
   }
+
+  checkHelpDisplay() {
+    return new Promise((resolve) => {
+      if (this.help) {
+        return this.ovhUserPref.getValue(this.getHelpPreferenceKey())
+          .then((value) => {
+            this.displayHelp = value && this.help;
```

Comment:
It tests `value` :

```javascript
this.displayHelp = value && this.help;
```

* In case of `value` is `true`, will affect `this.help` in `this.displayHelp`
* In case of `value` is `false`, will affect `false` in `this.displayHelp`